.clang-format

```diff
@@ -8,7 +8,6 @@ AllowAllParametersOfDeclarationOnNextLine: false
 AllowShortIfStatementsOnASingleLine: false
 AllowShortLoopsOnASingleLine: false
 AllowShortFunctionsOnASingleLine: None
-AllowShortLoopsOnASingleLine: false
```

Comment:
why is this flag dropped?

Reply:
It is a duplicate line and causes the yaml inter to fail.  Line 9 is the same option.